config/database.yml

```diff
@@ -58,6 +58,19 @@ development:
 test:
   <<: *default
   database: project_42_test
+  username: <%= ENV['development_username'] %>
```

Comment:
since you`re using the same username and password for production and tests, you can put this configuration inside the default block.